test/watch.js

```diff
@@ -21,13 +21,15 @@ describe('gulp', function() {
 
     var writeTimeout = 125; // Wait for it to get to the filesystem
     var writeFileWait = function(name, content, cb) {
-      if (!cb) cb = function() {};
+      if (!cb) { cb = function() {} };
       setTimeout(function() {
         fs.writeFile(name, content, cb);
       }, writeTimeout);
     };
 
+    // jscs: disable maximumLineLength
```

Comment:
Why not just disable this rule in the test files instead of writing it out every single time?